sdks/python/test-suites/portable/py35/build.gradle

```diff
@@ -29,4 +30,3 @@ task preCommitPy35() {
     dependsOn portableWordCountBatch
     dependsOn portableWordCountStreaming
 }
```

Comment:
empty line

Reply:
Thanks, probably removed by editor on autosave. I don't think we need them, so I'll remove them in other files touched by this PR.

Reply:
Ok, I'll add it back since now we get no new line at the end of file :)

Reply:
Thanks, I'll remove this in a follow up change since I'd like to keep the history that tests passed on this PR without running another seed job.

Reply:
Actually, that was just an extra new line, just in this file. no follow up changes needed.